app-backup/luckybackup/luckybackup-0.4.9.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="...a powerful, fast and reliable backup & sync tool"
```

Comment:
```diff
-SUMMARY="...a powerful, fast and reliable backup & sync tool"
+SUMMARY="A powerful, fast and reliable backup & sync tool"
```

`SUMMARY` should start with a capital letter, not ellipsis.

Reply:
@alaviss  as you suggested have been change.. please review again..